cfme/base/ssui.py

```diff
@@ -25,29 +24,28 @@
 from widgetastic_manageiq import SSUIVerticalNavigation
 
 
+class SSUISettingsNavDropdown(NavDropdown):
+    """iuser pficon-user to differentiate from help menu"""
+    ROOT = ('//nav'
+            '//li[.//a[contains(@class, "dropdown-toggle") and '
+            './/i[contains(@class, "{}")]] '
+            'and contains(@class, "dropdown")]'.format(PFIcon.icons.USER))
+
+
+class SSUIHelpNavDropdown(NavDropdown):
+    ROOT = '//nav//li[@ng-if="vm.permissions.helpMenu"]'
+
+
 class SSUIBaseLoggedInPage(View):
     """This page should be subclassed by any page that models any other page that is available as
     logged in.
     """
     flash = FlashMessages('.//div[@id="flash_msg_div"]')
-    help = NavDropdown('.//li[./a[@id="dropdownMenu1"]]')
+    help = SSUIHelpNavDropdown()
```

Comment:
isn't `help` a keyword in python?

Reply:
sure is, I was thinking it was just in ipython but its a built-in.  The way its used here I'm not too concerned, as it will never be called directly as a function.

 Okay leaving the changing of this to a non-keyword widget name in another PR? gotta audit for uses of it and change both UI and SSUI

Reply:
Please add a TODO :)

Reply:
https://github.com/ManageIQ/integration_tests/commit/203bc8ab79a4fbf9f7ea042fce346fa774fb6049#diff-7c97dce5eefb958f90c137c66bd8f857R18